deployments/helm-chart/templates/controller-appprotect-dos-arb.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.controller.appprotectdos.enable }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "nginx-ingress.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+    spec:
+      containers:
+        - name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+          image: "{{ .Values.controller.appprotectdos.arbitrator.repository }}"
```

Comment:
could we support the image related parameters in a consistent way with the IC?


```
  image:
    ## The image repository of the Ingress controller.
    repository: nginx/nginx-ingress

    ## The tag of the Ingress controller image.
    tag: "1.12.0"

    ## The pull policy for the Ingress controller image.
    pullPolicy: IfNotPresent
```

For example:
arbitrator.image.repository
arbitrator.image.tag
arbitrator.image.pullPolicy

Reply:
done